bin/php-scoper

```diff
@@ -10,17 +10,20 @@
  * file that was distributed with this source code.
  */
 
-use Webmozart\PhpScoper\Console\Application;
-use Webmozart\PhpScoper\Console\ApplicationConfig;
+use PackageVersions\Versions;
+use Symfony\Component\Console\Application;
+use Humbug\PhpScoper\Console\Command\AddPrefixCommand;
 
 if (file_exists($autoload = __DIR__.'/../../../autoload.php')) {
     require_once $autoload;
 } else {
     require_once __DIR__.'/../vendor/autoload.php';
 }
 
-$app = new Application();
-$conf = new ApplicationConfig();
-$conf->configure($app);
+$app = new Application('php-scoper', Versions::getVersion('humbug/php-scoper'));
```

Comment:
The output of `Versions::getVersion()` is `<version>@<hash>`, maybe we could stick to `<version>` if the version is a tagged one and `<hash>` otherwise.

For the test I think we should have one test somewhere with the complete output. That said, I think we should approach the tests differently. Indeed right now we are testing the output each time which has two drawbacks:

- It is tedious, especially when some dynamic parameters comes e.g. when you start to display "Too X time and Y MB"
- The output is the thing the most likely to change like any UI making the tests more painful as any change in the UI impacts a lot of tests
- It is not really what we want to test: we are interested in the output in a few tests, because that's what we want to test, otherwise we only want to test the scoping